src/components/releaseView/ReleaseView.jsx

```diff
@@ -89,12 +86,6 @@ const TablePaper = styled(Paper)`
   }
 `;
 
-// empty html tag | empty string
-const isEmptyRegEx = /<[^/][^>]*><\/[^>]+>|^$/;
-const hasText = str => {
-  return !(str ? isEmptyRegEx.test(str) : true);
-};
-
 ReleaseView.propTypes = {
   /*release: PropTypes.objectOf(
     (PropTypes.shape = {
```

Comment:
Kan den fjernes eller er det noe vi kanskje har bruk for i fremtiden?

Reply:
Ka du tenke på? Funksjonen blei flytta til `releaseNoteUtil.js`, sia den blir brukt fleire plassa

Reply:
Du har fortsatt ikke avslørt ka du tenkt på Jan Antonio Banderas..